fcli-core/fcli-fod/src/main/java/com/fortify/cli/fod/_common/util/FoDEnums.java

```diff
@@ -506,4 +506,104 @@ public static AttributeTypes fromInt(int val) {
         }
     }
 
+    public enum VulnerabilitySeverityType {
```

Comment:
I know similar field & method names are used in other enums in `FoDEnums`, but can we improve these to make it more clear what values are sent to/received from FoD, for example using field name `fodValue` and methods `getFoDValue` and `fromFoDValue`?